model/customlist.py

```diff
@@ -265,7 +269,8 @@ def set_work(self, metadata=None, metadata_client=None):
             # Try using the less reliable, more expensive method of
             # matching based on equivalent identifiers.
             equivalent_identifier_id_subquery = Identifier.recursively_equivalent_identifier_ids_query(
-                self.edition.primary_identifier.id, levels=3, threshold=0.5)
+                self.edition.primary_identifier.id, policy=policy
+            )
```

Comment:
Most of this diff is like this -- changing methods that call other methods so that they propagate (or create) a PresentationCalculationPolicy object instead of propagating (or making up) values for one or more of the three tuning variables.